components/brave_ads/browser/ads_tooltips_delegate.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_TOOLTIPS_DELEGATE_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_TOOLTIPS_DELEGATE_H_
+
+#include <string>
+
+#include "brave/components/brave_adaptive_captcha/buildflags/buildflags.h"
+
+namespace brave_ads {
+
+class AdsTooltipsDelegate {
+ public:
+  virtual ~AdsTooltipsDelegate() = default;
+
+#if BUILDFLAG(BRAVE_ADAPTIVE_CAPTCHA_ENABLED)
```

Comment:
this class seems kind of pointless if adaptive captcha is not enabled? Can we just put the whole thing in a gn guard?

Reply:
Yep, done.